site/AntDesign.Docs/Demos/Components/Slider/demo/Basic.razor

```diff
@@ -1,6 +1,6 @@
 ﻿<div>
     <Slider TValue="double" DefaultValue="35" Disabled="@_disabled"  />
-    <Slider TValue="(double, double)" DefaultValue="(20, 50)" Disabled="@_disabled" />
+    <Slider TValue="(double, double)" Value="_value" Min="15" Max="75" DefaultValue="_value" Disabled="@_disabled" />
```

Comment:
I don't think you should replace this. Line. But you can add it. Maybe this:
```razor
<div>
    <Slider TValue="double" DefaultValue="35" Disabled="@_disabled"  />
    <Slider TValue="(double, double)" DefaultValue="(20, 50)" Disabled="@_disabled" />
    <Slider TValue="(double, double)" Value="_value" Min="15" Max="75" DefaultValue="_value" Disabled="@_disabled" Marks=@_marks />

    Disabled: <Switch Size="small" Checked="@_disabled" OnChange="(e)=>OnSwitch(e)" />
</div>

@code
{

    private bool _disabled;

    private (double, double) _value = (15, 75);

    private SliderMark[] _marks =
        {
            new SliderMark(15, "15"),
            new SliderMark(75, "75")
    };

    private void OnSwitch(bool args)
    {
        _disabled = args;
    }
}
```
This though will also point you to this issue:
![image](https://user-images.githubusercontent.com/6518006/133433648-9163ed86-959a-41af-8285-b102db646f17.png)

Do you think you could fix it?

Reply:
@anddrzejb Yep, I will update the demo and look into the issue with the SliderMarks. Thanks!

Reply:
@anddrzejb I made those changes. Let me know if you have any other feedback. Thanks!